plugins/linux/ifplugin/descriptor/interface_veth.go

```diff
@@ -150,16 +149,6 @@ func getVethAlias(vethName, peerName string) string {
 	return vethName + "/" + peerName
 }
 
-// parseVethAlias parses out VETH logical name together with the peer name from the alias.
-func parseVethAlias(alias string) (vethName, peerName string) {
```

Comment:
Since `parseVethAlias` has been moved to `linuxcalls`, perhaps `getVethAlias` should be moved there as well (since they are inverse functions to each other). If the format for alias changes in the future for whatever reason, it would require code changes only in one place.
On the VPP side we use interface tags for the purpose of storing/dumping interface logical names and this is also completely hidden on the vppcalls side, and so with this change we would be also more consistent between vpp and linux plugins.
Similarly for `getTapAlias` and `parseTapAlias` (btw. I do not see `parseTapAlias` being removed from `interface_tap.go` in this diff)